fvm/state/state.go

```diff
@@ -198,18 +198,24 @@ func (s *State) MergeState(other *State) error {
 	return s.checkMaxInteraction()
 }
 
+type sortedAddresses []flow.Address
+
+func (a sortedAddresses) Len() int           { return len(a) }
+func (a sortedAddresses) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a sortedAddresses) Less(i, j int) bool { return bytes.Compare(a[i].Bytes(), a[j].Bytes()) >= 0 }
```

Comment:
One more tip is to use `a[i][:]` which should give us byte slice faster than calling separate function.